codespell_lib/data/dictionary.txt

```diff
@@ -10658,9 +10660,12 @@ plugable->pluggable
 pluign->plugin
 pluigns->plugins
 pluse->pulse
+plyotropy->pleiotropy
+pneumonic->mnemonic
 pobular->popular
 pobularity->popularity
 pocessed->processed, possessed,
+pocession->possession
```

Comment:
"procession" also?

Reply:
added

Reply:
I think procession should be first, as it's a smaller typo.

Reply:
Done